addons/binding/org.openhab.binding.jenocean/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you also rename the folder `addons/binding/org.openhab.binding.jenocean` to `addons/binding/org.openhab.binding.enocean`?